requirements.txt

```diff
@@ -1 +1,31 @@
 discord #for later
+
+# List of arguments in the database:
+0 - id
+1 - name
+2 - emoji
+3 - activity
+4 - channel
+5 - role
+6 - fakerole
+7 - uses
+8 - votes
+9 - threatened
+10 - enchanted
+11 - demonized
+12 - powdered
+13 - frozen
+14 - undead
+15 - bites
+16 - bitten
+17 - souls
+18 - lovers
+19 - sleepers
+20 - amulets
+21 - zombies
+
+
+# TODO
+- Add emoji checker in ./interpretation/check.py
+- Complete command list in ./interpretation/head.py
+- Add functinos for the command list above in ./roles_n_rules/functions.py
```

Comment:
This won't pass tests. Remove things that aren't dependencies.

Reply:
Is requirements.txt supposed to contain merely packages for installing?

Reply:
If so, where should I move this to?

Reply:
Yep, that's what requirements.txt is for. If you look in `.travis.yml`, you should see that it installs `pip` dependencies from here.